pkg/gui/list_context_config.go

```diff
@@ -69,7 +69,7 @@ func (gui *Gui) branchesListContext() IListContext {
 		OnRenderToMain:  OnFocusWrapper(gui.branchesRenderToMain),
 		Gui:             gui,
 		GetDisplayStrings: func(startIdx int, length int) [][]string {
-			return presentation.GetBranchListDisplayStrings(gui.State.Branches, gui.State.ScreenMode != SCREEN_NORMAL, gui.State.Modes.Diffing.Ref)
+			return presentation.GetBranchListDisplayStrings(gui.State.Branches, gui.State.ScreenMode != SCREEN_NORMAL, gui.State.Modes.Diffing.Ref, gui.Common.Tr)
```

Comment:
Because the `Common` attribute is so commonly used it's a nested struct, so the convention is to just say `gui.Tr`. Could we update that here (and elsewhere in the PR)?

Reply:
@john-nguyen09 I have created a new PR to your feature branch to fix a broken test and incorporate the change that @jesseduffield requested above: https://github.com/john-nguyen09/lazygit/pull/3

Reply:
Thanks @abid-mujtaba, that is great. Sorry, I am a little non-responsive, so I really appreciate your help.